source/winVersion.py

```diff
@@ -70,94 +136,32 @@ def __ge__(self, other):
-	"""Returns the public release name for a given Windows release based on major, minor, and build.
-	This is useful if release names are not defined as part of Windows version constants.
-	For example, 6.1 will return 'Windows 7'.
-	For Windows 10, feature update release name will be included.
-	On server systems, unless noted otherwise, client release names will be returned.
-	For example, 'Windows 10 1809' will be returned on Server 2019 systems.
-	"""
-	if (major, minor) == (6, 1):
-		return "Windows 7"
-	elif (major, minor) == (6, 2):
-		return "Windows 8"
-	elif (major, minor) == (6, 3):
-		return "Windows 8.1"
-	elif major == 10:
-		# From Version 1511 (build 10586), release Id/display version comes from Windows Registry.
-		# However there are builds with no release name (Version 1507/10240)
-		# or releases with different builds.
-		# Look these up first before asking Windows Registry.
-		if build in BUILDS_TO_RELEASE_NAMES:
-			return BUILDS_TO_RELEASE_NAMES[build]
-		# On Windows 10 1511 and later, cache the version in use on the system.
-		with winreg.OpenKey(
-			winreg.HKEY_LOCAL_MACHINE, r"Software\Microsoft\Windows NT\CurrentVersion"
-		) as currentVersion:
-			# Version 20H2 and later where a separate display version string is used.
-			try:
-				releaseId = winreg.QueryValueEx(currentVersion, "DisplayVersion")[0]
-			except OSError:
-				# Don't set anything if this is Windows 10 1507 or earlier.
-				try:
-					releaseId = winreg.QueryValueEx(currentVersion, "ReleaseID")[0]
-				except OSError:
-					releaseId = "unknown"
-		return f"Windows 10 {releaseId}"
-	else:
-		raise RuntimeError("Unknown Windows release")
+WIN7 = WinVersion(major=6, minor=1, build=7600)
+WIN7_SP1 = WinVersion(major=6, minor=1, build=7601, servicePack="1")
+WIN8 = WinVersion(major=6, minor=2, build=9200)
+WIN81 = WinVersion(major=6, minor=3, build=9600)
+WIN10 = WIN10_1507 = WinVersion(major=10, minor=0, build=10240)
+WIN10_1511 = WinVersion(major=10, minor=0, build=10586)
+WIN10_1607 = WinVersion(major=10, minor=0, build=14393)
+WIN10_1703 = WinVersion(major=10, minor=0, build=15063)
+WIN10_1709 = WinVersion(major=10, minor=0, build=16299)
+WIN10_1803 = WinVersion(major=10, minor=0, build=17134)
+WIN10_1809 = WinVersion(major=10, minor=0, build=17763)
+WIN10_1903 = WinVersion(major=10, minor=0, build=18362)
+WIN10_1909 = WinVersion(major=10, minor=0, build=18363)
+WIN10_2004 = WinVersion(major=10, minor=0, build=19041)
+WIN10_20H2 = WinVersion(major=10, minor=0, build=19042)
+WIN10_21H1 = WinVersion(major=10, minor=0, build=19043)
 
 
 def getWinVer():
 	"""Returns a record of current Windows version NVDA is running on.
 	"""
 	winVer = sys.getwindowsversion()
-	try:
-		releaseName = getWindowsReleaseName(major=winVer.major, minor=winVer.minor, build=winVer.build)
-	except RuntimeError:
-		releaseName = "Windows release unknown"
 	return WinVersion(
 		major=winVer.major,
 		minor=winVer.minor,
 		build=winVer.build,

```

Comment:
Since this getting the currently running version of windows:
```suggestion
	try:
		releaseName = f"Windows 10 {_getRunningVersionNameFromWinReg()}"
	except RuntimeError:
		releaseName = None
	return WinVersion(
		major=winVer.major,
		minor=winVer.minor,
		build=winVer.build,
		releaseName=releaseName,
```

Reply:
When preparing for #12585, we might as well enhance release name or the private WinReg function to say "Windows 11" if running specific builds.